config/pinpoint_overrides.yml

```diff
@@ -1,22 +1,92 @@
+AD:
+  translated_names:
+    en: Andorra
+    fr: Andorre
+    es: Andorra
```

Comment:
I love that we're adding these!

- My first thought is that these translated names are more like i18n data than the config data, so maybe we could organize it like that? We'd move these into like `config/i18n/countries/{en,fr,es}.yml` and then have  `PhoneNumberCapabilities` use `I18n` to look up the names?

- We could also consider removing the `name` from this overrides file, and pull that from the english I18n as a default to be backwards compatible with the old JSON format? (Name would still be required as part of the intermediate download of docs from pinpoint because we need name to merge sms + voice, since voice page doesn't use country codes)


LMK what you think of those approaches, it would be a bigger set of changes than what we have here currently